presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestWriter.java

```diff
@@ -375,13 +379,51 @@ public void testCollectColumnStatisticsOnInsert()
                         "('c_double', null, 0E0, 0E0, null, null, null, null), " +
                         "('c_timestamp', null, 0E0, 0E0, null, null, null, null), " +
                         "('c_varchar', 0E0, 0E0, 0E0, null, null, null, null), " +
+                        "('c_varbinary', null, 0E0, 0E0, null, null, null, null), " +
                         "('c_array', null, 0E0, 0E0, null, null, null, null), " +
                         "('p_varchar', 0E0, 0E0, 0E0, null, null, null, null), " +
-                        "(null, null, null, null, 0E0, null, null, null)) AS x (c_boolean, c_bigint, c_double, c_timestamp, c_varchar, c_array, p_varchar, p_varchar)");
+                        "(null, null, null, null, 0E0, null, null, null)) AS x (column_name, data_size, distinct_values_count, nulls_fraction, row_count, low_value, high_value, histogram)");
 
         dropTableIfExists(tmpTableName);
     }
 
+    @Test
+    public void testColumnFilter()
+    {
+        Session session = buildSessionForTableWrite();
```

Comment:
Don't think buildSessionForTableWrite is explicitly needed here. Did you try this SQL with a regular Session ?